src/harness/testcases/WINNF_FT_S_PPR_testcase.py

```diff
@@ -413,14 +413,13 @@ def generate_PPR_3_default_config(self, filename):
     """High-level description of the default config:
 
         SAS UUT has devices B, D; first gets a PAL grant and second is GAA.
-        SAS TH has devices A, C, E, none of which have a PAL grant.
+        SAS TH has devices A, C, E, all of which have a PAL grant.
 
         SAS UUT has one PPA, with device B as the only CBSD.
         SAS TH has one PPA, with only non-granted CBSDs.
```

Comment:
I see now your initial point :)
as you mentioned, you wanted the SAS TH grants to be GAA and not part of the PPA.
but as now they are PAL, this comment need an update.

Reply:
Thank you! I'm rushing too much...